### ISSUE TITLE
Issue with importing meshes with assigned textures

### ISSUE THREAD

**Issue by ICodeMaster** [ross-g/io_pdx_mesh#2]
![image](https://user-images.githubusercontent.com/4450978/62823427-9f925c80-bb5e-11e9-81b5-fd16c7d7a380.png)

Can't find texture path that exists

**Comment by ross-g** [comment]
Doesn't seem like a common problem, can you send me the file you're importing so I can take a look please?

The reason it's failing to find a texture path, is that it's trying to load a directory instead of a file path. Does the .mesh file work fine in game and only break when re-imported?

**Comment by ICodeMaster** [comment]
Indeed! It works in game, though the textures aren't properly assigned and I have to do it manually (only the first shader gets textures proper)

https://drive.google.com/file/d/19Lq19j16Hg22GfnPf2APn5yIpbLerFH6/view?usp=sharing

**Comment by ross-g** [comment]
So I've mostly tracked this down, but I need to check out a few things about how Clausewitz picks up textures for assets.

For now you can fix the issue by keeping all assigned textures in the same folder as the .mesh export and unticking the "Relative Path" option when picking textures in Blender.

**Comment by ICodeMaster** [comment]
Ok! That makes sense, I didn't realize it stored the texture path in the .mesh as well.
